task/buildah/0.1/buildah.yaml

```diff
@@ -96,8 +96,6 @@ spec:
       value: vfs
     - name: HERMETIC
       value: $(params.HERMETIC)
-    - name: PREFETCH_INPUT
-      value: $(params.PREFETCH_INPUT)
```

Comment:
Huh, today I learned that removing a param is not a breaking change, Tekton will just ignore unknown params :open_mouth: 

cc @lcarva, I think I saw that in your migration table somewhere

@scoheb could you also remove the param from the `template-build.yaml`?

Reply:
I did not remove the param. only the env var.

Reply:
Ohh, I see. But I actually tried passing something completely nonexistent to the buildah task, and it worked